server/notification-providers/wpush.js

```diff
@@ -0,0 +1,50 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class WPush extends NotificationProvider {
+    name = "WPush";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        const okMsg = "Sent Successfully.";
+
+        try {
+            const ret = await axios.post("https://api.wpush.cn/api/v1/send", {
+                "title": this.checkStatus(heartbeatJSON, monitorJSON),
+                "content": msg,
+                "apikey": notification.wpushAPIkey,
+                "channel": notification.wpushChannel
+            });
+            if (ret.data.code !== 0) {
+                return ret.data.message;
```

Comment:
If this is an error condition, maybe we should do `throw new Error(ret.data.message)` instead.